components/Note/src/note.css

```diff
@@ -0,0 +1,49 @@
+.denhaag-note {
```

Comment:
This file is named `note.css`. We're moving towards using `index.css` as a name for single-stylesheet components, since the directory itself already indicates the component.